templates/swiftstorage/config/00-account-server.conf

```diff
@@ -1,4 +1,5 @@
 [DEFAULT]
+bind_ip = {{ .BindIP }}
```

Comment:
This will likely break dataplane deployments  (same for all other services except the proxy-server)

Reply:
why would it? dataplane nodes would be using dns entries to talk to the services right?

Reply:
Because Swift is deployed on the dataplane as well and uses these as config files too

Reply:
Without this, those services start but they won't serve requests and fail with...

`RESP: [503] content-length: 118 content-type: text/html; charset=UTF-8 date: Tue, 05 Mar 2024 12:11:57 GMT server: Apache set-cookie: 8555ec7bc3b761f9a531b621867c3563=474f3856f1468e7a0f973a8830e54de5; path=/; HttpOnly; Secure; SameSite=None x-openstack-request-id: txe5a0ba05928e4c23b5723-0065e70c0d x-trans-id: txe5a0ba05928e4c23b5723-0065e70c0d
RESP BODY: Omitted, Content-Type is set to text/html; charset=UTF-8. Only text/plain, application/json responses have their bodies logged.
GET call to https://swift-public-openstack.apps.sno.lab.example.com/v1/AUTH_21f6f4ba3f444e3a94dd4de56d7173c3?format=json used request id txe5a0ba05928e4c23b5723-0065e70c0d
Request returned failure status: 503
Service Unavailable (HTTP 503) (Request-ID: txe5a0ba05928e4c23b5723-0065e70c0d)
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/cliff/app.py", line 410, in run_subcommand
    result = cmd.run(parsed_args)
  File "/usr/lib/python3.9/site-packages/osc_lib/command/command.py", line 39, in run
    return super(Command, self).run(parsed_args)
  File "/usr/lib/python3.9/site-packages/cliff/display.py", line 117, in run
    column_names, data = self.take_action(parsed_args)
  File "/usr/lib/python3.9/site-packages/openstackclient/object/v1/container.py", line 171, in take_action
    data = self.app.client_manager.object_store.container_list(
  File "/usr/lib/python3.9/site-packages/openstackclient/api/object_store_v1.py", line 146, in container_list
    return self.list('', **params)
  File "/usr/lib/python3.9/site-packages/openstackclient/api/api.py", line 192, in list
    ret = self._request(
  File "/usr/lib/python3.9/site-packages/openstackclient/api/api.py", line 83, in _request
    return session.request(url, method, **kwargs)
  File "/usr/lib/python3.9/site-packages/keystoneauth1/session.py", line 986, in request
    raise exceptions.from_response(resp, method, url)
keystoneauth1.exceptions.http.ServiceUnavailable: Service Unavailable (HTTP 503) (Request-ID: txe5a0ba05928e4c23b5723-0065e70c0d)
clean_up ListContainer: Service Unavailable (HTTP 503) (Request-ID: txe5a0ba05928e4c23b5723-0065e70c0d)
END return value: 1
`

Reply:
Where does this failure happens? The service starts fine in all the tests I've seen until now

Reply:
> Where does this failure happens? The service starts fine in all the tests I've seen until now

Are you testing with ipv6 cluster? You can't use the default 0.0.0.0 in ipv6 clusters.

Reply:
Sorry, reviewed the patch again. I do see why this failed on the error on IPv6 now.

So this would most likely work if the dataplane nodes also use IPv6. Do we assume this as given? If so, it should be fine, but probably I need to have a closer look

Reply:
Yes. single stack ipv6 means both controlplane and dataplane both use ipv6. We don't have plans to support dual stack now.